apps/coscrad-frontend/src/components/resources/utils/render-transcribed-audio-media-cell.tsx

```diff
@@ -0,0 +1,7 @@
+import { MediaPlayer } from '@coscrad/media-player';
+
+export const renderTranscribedAudioMediaCell = (url: string): JSX.Element => (
+    <div className="transcribed-audio-player" title={url}>
+        <MediaPlayer audioUrl={url} />
```

Comment:
Nice! Is this branch broken until the MediaPlayer updates? Let's talk about whether to do that now or make it a separate story (as you drafted so far).

Reply:
Ok, yes, let's make a plan.

Reply:
We have a story that we will collaborate on to address this.